app/views/active_waiter/jobs/_reload.html.erb

```diff
@@ -1,3 +1,3 @@
 <script>
-  setTimeout(function() { window.location.href = <%= url_for(params.merge(retries: @retries)).to_json.html_safe %>; }, 2000);
+  setTimeout(function() { window.location.href = <%= url_for(params.permit!.merge(retries: @retries)).to_json.html_safe %>; }, 2000);
```

Comment:
This may have security issue.
http://eileencodes.com/posts/actioncontroller-parameters-now-returns-an-object-instead-of-a-hash/